autopts/wid/mcp.py

```diff
@@ -287,6 +298,8 @@ def hdl_wid_20107(params: WIDParams):
     elif "Current Track Object" in params.description:
         btp.mcp_current_track_obj_id_read(addr_type, addr)
         ev = stack.mcp.wait_current_track_obj_id_ev(addr_type, addr, 10)
+        if ev is not None:
+            stack.mcp.object_id = ev[3]
```

Comment:
Why type of object is `ev` here? Where is it documented that the ID is in `ev[3]`?

Reply:
this is just python tuple, there are no dedicated classes for all those events

https://github.com/auto-pts/auto-pts/blob/master/autopts/pybtp/btp/mcp.py#L729

Reply:
Thanks :)